pkg/hubble/container/ring_test.go

```diff
@@ -562,6 +562,7 @@ func TestRing_ReadFrom_Test_1(t *testing.T) {
 		t,
 		// ignore go routines started by the redirect we do from klog to logrus
 		goleak.IgnoreTopFunction("k8s.io/klog.(*loggingT).flushDaemon"),
+		goleak.IgnoreTopFunction("k8s.io/klog/v2.(*loggingT).flushDaemon"),
```

Comment:
If `k8s.io/klog` was replaced with `k8s.io/klog/v2`, like it seems to be the case, we can delete the lines referring `k8s.io/klog`, right?

Reply:
`klog` is still referenced in the vendor directory so I'll leave it for now until it disappears.